types/aws-lambda/trigger/cloudformation-custom-resource.d.ts

```diff
@@ -1,70 +1,85 @@
 import { Handler } from "../handler";
 
+export interface ResourcePropertiesCommon extends Record<string, any> {
```

Comment:
Everything is exported at the package top level, so all types should be qualified by their trigger

Reply:
Good catch - I've updated it in the latest. It came out a little awkward since the word `Resource` appears twice but I rationalize it that this is a supporting type and an end-user shouldn't be commonly using it. But happy to hear what you think if you have a different vision!